index.bs

```diff
@@ -1219,6 +1219,15 @@ access to that data in a {{Window}} global.
 
 Issue: Define how this works.
 
+# User Contexts # {#user-contexts}
+
+A <dfn>user context</dfn> is an aggregation of browsing contexts that separates the browser state for browsing contexts in one user context from browsing contexts in another user context.
+The user agent contains at least one user context, the <dfn>default user context</dfn>. 
+
+Note: User contexts are a WebDriver-specific abstraction to allow working with private browsing modes (https://www.w3.org/2001/tag/doc/private-browsing-modes/).
```

Comment:
This note seems inaccurate. User contexts may or may not map to private browsing (I wonder if we should consider a notion of user context flags, with "private" being one such flag? Certainly gecko containers _aren't_ like private browsing by default (in that they persist state to disk).

However we _should_ define that seperate user contexts don't share state (e.g. cookies, storage, etc.)

Reply:
Updated based on your suggestion on chat!